Assets/MixedRealityToolkit/Providers/UnityInput/MouseDeviceManager.cs

```diff
@@ -90,12 +134,13 @@ public override void Enable()
             }
 
             Controller.SetupConfiguration(typeof(MouseController));
-            inputSystem?.RaiseSourceDetected(Controller.InputSource, Controller);
+            InputSystem?.RaiseSourceDetected(Controller.InputSource, Controller);
         }
 
         /// <inheritdoc />
         public override void Update()
         {
+            // If a mouse is present, and we do not yet
```

Comment:
>// If a mouse is present, and we do not yet [](start = 12, length = 43)

I'm waiting with bated breath

We do not yet?!!?

Reply:
haha! i mustve lost my train of thought on that comment :)

Reply:
removed as it wasnt going to add much value